staff/carme-casserras/array-functions/lastindexof.js

```diff
@@ -0,0 +1,23 @@
+'user strict'
+
+/**
+ * Returns the last index at which a given element can be found in the array, or -1 
+ * 
+ * @param {Array} array The array to evaluate.
+ * 
+ * 
+ */
+
+function lastIndexOf(array, elem) {
+    if (!(array instanceof Array)) throw TypeError(array + ' is not an array')
+    
+    var acum = 0;
+    for (var i = 0; i < array.length; i++) {
```

Comment:
Deberías recorrer el array desde atrás hacía adelante. 
la fórmula normal es
for (var i = array.length -1; i >= 0; i--)

con esto lo recorres desde el último al primero